deeppavlov/core/data/simple_vocab.py

```diff
@@ -48,9 +50,11 @@ def __init__(self,
         if self.load_path:
             self.load()
 
-    def fit(self, tokens):
+    def fit(self, *args):
```

Comment:
Why args?

Reply:
Right. To be able to pass several variables to `fit()`.